core/input/keyboard.js

```diff
@@ -307,7 +307,7 @@ export default class Keyboard {
 
         const target = this._target;
         const downList = this._keyDownList;
-        ['AltLeft', 'AltRight'].forEach(function (code) {
+        ['AltLeft', 'AltRight'].forEach((code) => {
```

Comment:
In the other places you removed the parentheses around the parameter:

    code => { ... }

However, I think I prefer leaving the parentheses in like this, but it should at least be consistent.